src/core/qgsproviderregistry.h

```diff
@@ -98,7 +98,7 @@ class CORE_EXPORT QgsProviderRegistry
      * responsible for deleting the returned widget.
      */
     QWidget *createSelectionWidget( const QString &providerKey,
-                                    QWidget *parent = nullptr, Qt::WindowFlags fl = Qt::WindowFlags() );
+                                    QWidget *parent = nullptr, Qt::WindowFlags fl = Qt::WindowFlags(), bool embeddedMode = false );
```

Comment:
Done in https://github.com/qgis/QGIS/pull/4629/commits/8564e8b2b65c7b5aec379a7c5dfb0057076936e7